pyproject.toml

```diff
@@ -144,10 +144,6 @@ sections = [
     "MEDPHYS", "FIRSTPARTY", "LOCALFOLDER"
 ]
 
-[tool.dephell.main]
```

Comment:
This doesn't seem related to pinning the images.
While it's not an issue, it wasn't clear from the top level commit line

Reply:
Yeah good cool. I was doing a CTRL+F for main and master and that config popped up. It's old config that isn't doing anything. Figured a fly by removal wasn't an issue in this case.

Reply:
It's not an issue.
But it could have been if it were accidental.
So I thought I would mention it without preventing progress

Reply:
Yup good call.